### PR TITLE
EDU-567 Expand files view components

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-567

Screen Shot 2022-05-17 at 17 06 10

### CODE REVIEW

**Comment by ahelmberger** [comment]
> Closes https://educandu.atlassian.net/browse/EDU-567
> 
> Screen Shot 2022-05-17 at 17 06 10

As a suggestion: the selection looks somewhat different from the selections we have for example in the plugin selector or the UI language selector, maybe we could adjust the styling a little bit to blend in more nicely?

localhost_3000_lessons_wjqSKgjiVoTjRWkFaagQfR_view=edit

**Comment by adrianaluchian** [comment]
> > Closes https://educandu.atlassian.net/browse/EDU-567
> > [...]
> 
> As a suggestion: the selection in the grid view looks somewhat different from the selections we have for example in the plugin selector or the UI language selector, maybe we could adjust the styling a little bit to blend in more nicely?
> 
> [...]
>

I was trying to somehow match it to the `files-list-view` selection, but I changed it towards the plugin selection and it looks indeed better:

Screen Shot 2022-05-17 at 17 41 08

**Comment by ahelmberger** [comment]
> > > Closes https://educandu.atlassian.net/browse/EDU-567
> > > [...]
> > 
> > 
> > As a suggestion: the selection in the grid view looks somewhat different from the selections we have for example in the plugin selector or the UI language selector, maybe we could adjust the styling a little bit to blend in more nicely?
> > [...]
> 
> I was trying to somehow match it to the `files-list-view` selection, but I changed it towards the plugin selection and it looks indeed better:
> 
> Screen Shot 2022-05-17 at 17 41 08

Nice, looks good :heart_eyes:

File: src/resources/resources.json [hunk 1]
```diff
@@ -1958,6 +1958,8 @@
       "audioResource": "audio",
       "textResource": "text",
       "pdfResource": "document",
+      "noneResource": "Unknown",
```
[the remark this answers is not in the document]
Comment (ahelmberger) [reply src/resources/resources.json]: Cool, thanks!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**